core/lib/multivm/src/glue/types/vm/vm_partial_execution_result.rs

```diff
@@ -36,9 +36,11 @@ impl GlueFrom<crate::vm_m6::vm_instance::VmPartialExecutionResult>
             statistics: crate::interface::VmExecutionStatistics {
                 contracts_used: value.contracts_used,
                 cycles_used: value.cycles_used,
-                gas_used: value.computational_gas_used,
```

Comment:
It was before, why it's zero now?

Reply:
I think it was a bug before, I decided to explicitly pass 0 instead of "something"